Content.Shared/_Impstation/Construction/Steps/RemoveEntityConstructionGraphStep.cs

```diff
@@ -0,0 +1,50 @@
+using Content.Shared.Construction;
+using Content.Shared.Construction.Steps;
+using Content.Shared.Examine;
+using Content.Shared.Tag;
+
+namespace Content.Shared._Impstation.Construction.Steps;
+
+[DataDefinition]
+public sealed partial class EntityRemoveConstructionGraphStep : ConstructionGraphStep
+{
+    /// <summary>
+    /// A tag of the item you want to remove.
+    /// </summary>
+    [DataField("remove")]
+    private string? _tag;
+
+    /// <summary>
+    /// A string representing the '$name' variable of the Loc file. By default, it's "Next, remove {$name}".
+    /// </summary>
+    [DataField("name")]
```

Comment:
forgot to note with the last review but the tag here (`"name"`) is redundant, datafields automatically inherit the name of their variable. for example, `GuideString` below could be defined in yaml with `guideString` without having to put that as arguments next to it